magento/magento2
# Add Note to Explain Search Weight Values

### Description (*)
Adds a note beneath the `Search Weight` attribute's `Storefront Properties` input to explain what the values mean.
+ See: https://docs.magento.com/user-guide/catalog/search-weighted.html#set-the-search-weight-properties-of-an-attribute

### Manual testing scenarios (*)
1. Log into the admin
2. Navigate to `Stores -> Attributes -> Product`
3. Select any attribute from the grid
4. Open the `Storefront Properties` options on the left hand side
5. Ensure `Use in Search` is enabled
6. Observe the note is present beneath the `Search Weight` option
7. The note should read: `10 is the highest priority/heaviest weighting.`

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#36646: Add Note to Explain Search Weight Values
